src/path/mod.rs

```diff
@@ -120,6 +144,44 @@ impl Path {
         })
     }
 
+    /// Convert a filesystem path to a [`Path`] relative to the filesystem root
+    ///
+    /// This will return an error if the path does not exist, or contains illegal
+    /// character sequences as defined by [`Path::parse`]
+    pub fn from_filesystem_path(path: impl AsRef<std::path::Path>) -> Result<Self, Error> {
+        Self::from_filesystem_path_with_base(path, None)
+    }
+
+    /// Convert a filesystem path to a [`Path`] relative to the provided base
+    ///
+    /// This will return an error if the path does not exist, contains illegal
+    /// character sequences as defined by [`Path::parse`], or does not start
```

Comment:
I found "provided prefix" confusing as I didn't know what it was referring to. It seems to  mean that the `path` must be prefixed by the the path in the `base` URL, if specified

Reply:
Oh sorry, I renamed the parameters and forgot to update the docstring